### PR TITLE
Update stuff a bit

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,38 +165,46 @@ function. In this way, the context can be used to override view functions.
...
+2> TFun = mustache:compile(simple).
+3> mustache:render(simple, TFun, Ctx).
+```
 
 This will produce the following output:
 
-    Hello Chris
-    You have just won $10000!
-    Well, $6000.00, after taxes.
+```erlang
```
Comment (g-andrade) [inline README.md]: ^ Should this be `text`?

File: benchmarks/bench.erl [hunk 2]
```diff
@@ -10,20 +10,20 @@ run() ->
...
   % Ctx1 = dict:new(),
-  CT = mustache:compile(complex, "../examples/complex.mustache"),
-  T0 = erlang:now(),
+  CT = mustache:compile(complex, "examples/complex.mustache"),
+  T0 = erlang:monotonic_time(),
   render(CT, Ctx1, ?COUNT),
-  T1 = erlang:now(),
-  Diff = timer:now_diff(T1, T0),
+  T1 = erlang:monotonic_time(),
+  Diff = erlang:convert_time_unit(T1 - T0, native, micro_seconds),
```
Comment (g-andrade) [inline benchmarks/bench.erl]: ^ No need for the legacy unit here (`micro_seconds`) - OTP 19, the new minimum required version as per this PR, deprecated those units and introduced the prettier ones (e.g. `microsecond`.)

**Review by g-andrade — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (paulo-ferraz-oliveira) [reply README.md]: Sure.

File: benchmarks/bench.erl (hunk 2 above)
Comment (paulo-ferraz-oliveira) [reply benchmarks/bench.erl]: Cool. Thanks.

**Review by g-andrade — APPROVED** [review]

**Pull request merged**